db/pgstorage/migrations/0006.sql

```diff
@@ -17,5 +21,9 @@ CREATE TABLE IF NOT EXISTS mt.rht_temp AS (SELECT key, min(value), max(deposit_i
 
 DROP INDEX IF EXISTS sync.claim_block_id;
 DROP INDEX IF EXISTS sync.deposit_block_id;
-DROP INDEX IF EXISTS sync.monitored_txs_block_id;
-DROP INDEX IF EXISTS sync.token_wrapped_block_id;
\ No newline at end of file
+DROP INDEX IF EXISTS sync.token_wrapped_block_id;
+
+ALTER TABLE sync.monitored_txs
+ADD COLUMN block_id BIGINT REFERENCES sync.block (id) ON DELETE CASCADE;
```

Comment:
Could it be migrated down? because the data in this column is not going to be filled... so if you undo this change is going to break the server